### ISSUE TITLE
the rm train model doesnot save value_head.bin, which is needed by the rm_engine function.

### ISSUE THREAD

**Issue by PanYicheng** [microsoft/rStar#19]
After training rm, the value head weight is stored inside the model.safetensors. while in the following rm_engine function, it reads an seperate file named value_head.bin.

`
def rm_engine(config):
    if config.need_value_func:
        prm_model = LLM(
            model=config.reward_model_dir, 
            task="reward",
            tensor_parallel_size=1, 
            trust_remote_code=True,
            max_model_len=config.max_model_len,
            enforce_eager=True,
            swap_space=0,
            gpu_memory_utilization=0.98 - config.llm_gpu_memory_utilization, # for qwen 7b, rm need 15G memory
        )
        
        v_head_state = torch.load(os.path.join(config.reward_model_dir, "value_head.bin"), weights_only=True)
        v_state = {}
        for name, param in v_head_state.items():
            v_state[name.replace("v_head.", "")] = param
        model_config = AutoConfig.from_pretrained(config.reward_model_dir, trust_remote_code=True, use_cache = False)
        v_head = ValueHead(model_config)
        v_head.load_state_dict(v_state)
        v_head.eval()

        tokenizer = AutoTokenizer.from_pretrained(config.reward_model_dir, trust_remote_code=True, use_cache = False, split_special_tokens=False,)
        return prm_model, v_head, tokenizer
    else:
        return None, None, None`

**Comment by Gxy-2001** [comment]
Thank you for your interest. I have uploaded save_rm.py, which serves as the script for converting RM format files. Let us know if you have any further questions!

https://github.com/microsoft/rStar/pull/22
